src/components/editTestCase/qdm/LeftPanel/ElementsTab/Elements/DataElementsCard/Codes/Codes.tsx

```diff
@@ -232,13 +299,27 @@ const Codes = ({
         )}
         <div tw="w-1/4 py-6">
           <IconButton
+            disabled={isFormValid()}
             data-testid="add-code-concept-button"
             onClick={addNewCode}
           >
             <AddCircleOutlineIcon sx={{ color: "#0073c8" }} />
           </IconButton>
         </div>
       </div>
+      <div tw="flex flex-wrap gap-2" id="these are chips">
```

Comment:
an `id` attribute should be without space

Reply:
ooh thats for my testing, forgot to remove.